# Define the request part of the service
---
# Define the response part of the service
sensor_msgs/PointCloud2 pointcloud2
